starter-project-web-react/components/Button.jsx

```diff
@@ -0,0 +1,60 @@
+import * as React from 'react';
```

Comment:
better to not name your component just Button, because different people may have different buttons that they have customized

It is also better to have it inside a folder that concerns your feature (i would suggest /components/blog/{new component name})

Reply:
I will do that.

Reply:
done